modules/spo-enabling-prometheus.adoc

```diff
@@ -0,0 +1,55 @@
+// Module included in the following assemblies:
+//
+// * security/security_profiles_operator/spo-advanced.adoc
+
+:_content-type: PROCEDURE
+[id="spo-enabling-prometheus_{context}"]
+= Enabling monitoring with Prometheus
+
+If the {product-title} cluster has Prometheus deployed, the Security Profiles Operator will automatically create a `ServiceMonitor` resource within the `openshift-security-profiles` namespace. This monitor resource allows automatic metrics discovery within the cluster.
+
+.Procedure
```

Comment:
Absolutely my fault again, because I didn't understand how metrics are supposed to be set up in OCP, at least properly. Since we install SPO into an `openshift-` namespace, we need to make sure that metrics work out of the box and this procedure should not be needed. Currently there is a patch upstream to fix the metrics at https://github.com/kubernetes-sigs/security-profiles-operator/pull/1358 once the patch is merged, we could remove this procedure, so adding the ConfigMap should not be needed as well as the `oc port-forward -n openshift-user-workload-monitoring pod/prometheus-user-workload-0 9090` (because OCP should not be considered a user-workload if shipped as part of OCP).

I'm sorry that I didn't realize this was in fact an OCP bug earlier!

Reply:
No problem, I'll remove the procedure!